src/components/InstructionsDrumMachine.js

```diff
@@ -0,0 +1,207 @@
+import { useState } from 'react';
+import styled from 'styled-components';
+
+import EQ from '../images/EQ.svg';
+import recordingLogo from '../images/recording-page.svg';
+import sequencerLogo from '../images/sequencer.svg';
+
+import { useNavigate } from 'react-router-dom';
+
+export default function InstructionsDrumMachine() {
+  let navigate = useNavigate('');
+  const [firstStepsVisible, setFirstStepsVisible] = useState(false);
+  const [secondStepsVisible, setSecondStepsVisible] = useState(false);
+  const [isNavVisible, setIsNavVisible] = useState(false);
+  const [isPopUpVisible, setIsPopUpVisible] = useState(true);
+  return (
+    <>
+      <PopUp visible={isPopUpVisible}>
+        New? <button onClick={instructionClick}>click for Instructions</button>
+        <button onClick={() => setIsPopUpVisible(false)}>X</button>
+      </PopUp>
+      <InstructionOne visible={firstStepsVisible}>
+        click on <br />
+        <img src={EQ} alt="volume-settings" height="25px" width="25px" />
+        <br />
+        to change the volume
+      </InstructionOne>
+      <InstructionTwo visible={firstStepsVisible}>
+        click on a pad to play a sample
+      </InstructionTwo>
+      <InstructionThree visible={firstStepsVisible}>
+        choose a Drum-Loop an click play to have a background sound
+      </InstructionThree>
+      <InstructionFour visible={firstStepsVisible}>
+        click here to record your session
+      </InstructionFour>
+      <InstructionFive visible={secondStepsVisible}>
+        {' '}
+        click on <br />
+        <img src={sequencerLogo} alt="recording" height="25px" width="25px" />
+        <br /> to get to the sequencer page
+      </InstructionFive>
+      <InstructionSix visible={secondStepsVisible}>
+        click on <br />
+        <img src={recordingLogo} alt="recording" height="25px" width="25px" />
+        <br /> to get to the recordings page
+      </InstructionSix>
+      <InstructionSeven visible={secondStepsVisible}>
+        click on <br />
+        <img src={recordingLogo} alt="recording" height="25px" width="25px" />
+        <br /> to get to the recordings page
+      </InstructionSeven>
+
+      <ExitButton visible={isNavVisible} onClick={exitClick}>
+        {' '}
+        EXIT
+      </ExitButton>
+      <NextButton visible={isNavVisible} onClick={nextClick}>
+        {' '}
+        NEXT
+      </NextButton>
+    </>
+  );
+  function nextClick() {
+    if (firstStepsVisible === true) {
+      setFirstStepsVisible(false);
+      setSecondStepsVisible(true);
+    } else {
+      navigate('/settings', { replace: true });
```

Comment:
The user gets to the settings after watching the second page of instructions? Do you plan to give the user there new instructions in a next user story? Otherwise I would wonder why to get to the settings then as a user.

Reply:
the user story is not finished yet. There will come more  😉